src/interfaces/GeoChart.ts

```diff
@@ -0,0 +1,5 @@
+// (C) 2019 GoodData Corporation
+export interface ICoordinates {
+    longtitude: number;
+    lattitude: number;
```

Comment:
longtitude -> longitude
lattitude -> latitude
They are my typos in google docs